certified-connectors/Airmeet/apiProperties.json

```diff
@@ -53,6 +53,7 @@
       }
     },
     "iconBrandColor": "#0A090B",
+    "scriptOperations": [],
```

Comment:
@amit-vb having an empty scriptOperations is not allowed. please define a proper scriptOperations property.
Please see [here ](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/Adobe%20Experience%20Manager/apiProperties.json#:~:text=%22scriptOperations%22%3A%20%5B,%5D%2C)for example.

Reply:
Hi @Amjed-Ayoub 
added & Thank you!